date: 2002/06/11 22:16:43;  author: jer;  state: Exp;  lines: +1 -1
fixing two old bugs,
pth 1.4.1 will now work correctly, we force the socket into blocking mode (thanks to Miguel Gomard <m.gomard@ilius.net>)
high ports for SRV work as well, using 6 bytes as allocated (thanks to "Safford, Brian" <brian.safford@eds.com>)

Index: dnsrv/srv_resolv.c
===================================================================
RCS file: /home/cvs/jabberd14/dnsrv/srv_resolv.c,v
retrieving revision 1.9
retrieving revision 1.10
diff -r1.9 -r1.10
73c73
<      snprintf(result, 5, "%d", port);
---
>      snprintf(result, 6, "%d", port);
